Java/search/TernarySearch.java

```diff
@@ -0,0 +1,89 @@
+import java.util.*;
+public class Main
+{  
+   // Recursive Approach
+  static int ternarySearchRecursive (int arr[], int f, int l, int h)
```

Comment:
Use variable names ```left``` and ```right``` instead of l and h. This will help others in better understanding the code.